NewSpring/crete
# perf: Improved Performance of Incrementing Prayers

## DESCRIPTION

### What does this PR do, or why is it needed?

This PR updates the createInteraction query - particularly the subquery that checks to see when the last notification was sent to the user - to use the lava endpoint, which in this context is more performant (~13x faster).

### How do I test this PR?

Running the api locally, run the `incrementPrayerCount` mutation using current code vs new code (I used postman to actually see duration of request/response).

```
mutation {
  incrementPrayerCount(nodeId: "Prayer:60d5cb69c96378192c2e12d5cda5a03d") {
    text
  }
}
```


## TODO


## REVIEW


---

> The purpose of a PR Review is to _improve the quality of the software._
